### ISSUE TITLE
Expand Multi-Check Functionality

### ISSUE THREAD

**Issue by yorkshirelandscape** [foundryvtt/pf2e#12729]
Could we expand on this to allow distinct names and traits per check? #7928 

I'd like to be able to do this: `@Check[type:diplomacy,society|name:Gather Information,Recall Knowledge|dc:15|adjustment:0,2|traits:action:gather-information,action:recall-knowledge]` or something to that effect. If this is too unwieldy, some other approach is fine. The goal, ultimately, is to be able to easily send multiple checks to chat with their own distinct names and traits. However that is accomplished is fine by me.

**Comment by yorkshirelandscape** [comment]
As long as I'm at it, I might as well throw in a request to make GM rolls as a trait. Would be nice for passive perception checks.

**Comment by TMunsie** [comment]
> As long as I'm at it, I might as well throw in a request to make GM rolls as a trait. Would be nice for passive perception checks.

Adding secret as a trait already does this, doesn’t it?

**Comment by yorkshirelandscape** [comment]
I was imagining a completely hidden roll. I always feel a little weird rolling blind perception for a bunch of players and then saying nothing. Maybe that's just me, though.

**Comment by TMunsie** [comment]
So you don’t want a GM roll? You want a roll that the players don’t know about. Thats not something that Foundry supports natively and you need a module for and we won’t build syntax for that until Foundry does it natively.

if you mean something else, please finish the sentence: “At an in person table I would…”

As a system we don’t try to override what the core software natively does unless there is a specific rule we’re handling from the rule book. Our position is that modules should override VTT design decisions that don’t impact how PF2e itself is played.

**Comment by yorkshirelandscape** [comment]
Oh, perhaps I'm not realizing what players actually see. I thought that a "Self Roll" was a roll that only the roller sees. Not so? I think of the pink rolls as secret and the blue ones and self rolls, but I could be sorely mistaken.

**Comment by Drental** [comment]
every roll in every roll mode is visible for all players - only the results are hidden for most of them.

Anything that makes messages not display at all is a module.

**Comment by yorkshirelandscape** [comment]
Fair enough. Please disregard the additional request, then. I don't think I have a module that's doing that. I was just making erroneous assumptions about what is and isn't visible.

**Comment by bogga** [comment]
I use Actually Secret Rolls to hide the chat messages for private rolls from my players (rather than just the default, native behaviour of making the roll type & result hidden). I do this because I find it declutters the chat for my players and also because I am sneaky
